### PR TITLE
update:link and added readme

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
feature added:readme and changed redirection link

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+## [ProductListingPage](https://drinks-break.netlify.app/productlisting/productlisting)
+
+Product Listing page consist of filter and listed product .Listed product changes on selection of filter.
+
+## [CartPage](https://drinks-break.netlify.app/cartmanagement/cart)
+
+CartPage consist of products added in the cart .listed products can be removed from the cart and added back to the whishList page.
+
+## [WhishListpage](https://drinks-break.netlify.app/wishlist/wishlist)
```
Comment (TejasShekar) [inline README.md:21]: *Wishlist Page

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+
+```diff
+Go and have a look of website
+```
+
+## [Landing Page/Home page](https://drinks-break.netlify.app/)
+
+This page consist of navbar and banner with a button which redirect to productListed page.There are listed categories clicking on These categories will redirect to product page with selcted categories.
+
+## [ProductListingPage](https://drinks-break.netlify.app/productlisting/productlisting)
```
Comment (TejasShekar) [inline README.md:13]: give space between words

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+
+user can login into application once completed successful signup.
+
+## [sign-out/Log-out](https://drinks-break.netlify.app/authentication/logout/logout)
+
+use can logout from the application after successful login
+
+## [sign-up](https://drinks-break.netlify.app/authentication/signup/signup)
+
+user can registed himself if he is the fresh user.
```
Comment (TejasShekar) [inline README.md:37]: "new user" would be a better instead of "fresh user".

**Review by TejasShekar — APPROVED** [review]
Suggested few grammatical changes. Rest looks good to me. 🍻

**Review by anshtripathi01 — APPROVED** [review]
Good to merge

**Comment by sk-ip** [comment]
looks good to me

**Pull request merged**